packages/fxa-graphql-api/src/gql/dto/payload/finished-setup-account.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class FinishedSetupAccountPayload {
+  @Field({
+    description: 'A unique identifier for the client performing the mutation.',
+    nullable: true,
+  })
+  public clientMutationId?: string;
+
+  @Field()
+  public uid!: hexstring;
+
+  @Field()
+  public sessionToken!: hexstring;
+
+  @Field()
+  verified!: boolean;
```

Comment:
I didn't see this value being returned in https://github.com/mozilla/fxa/blob/1cd01630d46d7dd98704ccd8540b25d6b914e02b/packages/fxa-auth-server/lib/routes/account.ts#L665. Is it used else where?

Reply:
That `verified` is the one here: https://github.com/mozilla/fxa/blob/1cd01630d46d7dd98704ccd8540b25d6b914e02b/packages/fxa-auth-server/lib/routes/account.ts#L630

Reply:
@vbudhram I don't quite understand your comment.  The line you linked is in the `setPassword` function, which afaict is not related to this PR.